app/components/utils/add-existing-piece.hbs

```diff
@@ -29,7 +27,7 @@
         @size={{this.size}}
         @noDataMessage={{t "no-results-found"}}
         @isLoading={{this.findAll.isRunning}}
-        @onClickRow={{action "select"}}
+        @onClickRow={{this.select}}
```

Comment:
Can we refactor the `select` -action in the component such that it doesn't set a `selected` flag on the piece recorde, but for example keeps a tracked property of selected piece(s)?.

Also rename the action arguments `add` and `delete` to `@onAdd` and `@onDelete`, or even more clear `@onLink` and `@onUnlink`.